hrms/patches/v14_0/update_loan_repayment_repay_from_salary.py

```diff
@@ -0,0 +1,10 @@
+import frappe
+
+
+def execute():
+	if frappe.db.exists("Custom Field", {"name": "Loan Repayment-repay_from_salary"}):
```

Comment:
```suggestion
	if frappe.db.exists("Custom Field", "Loan Repayment-repay_from_salary"):
```
Since it is the name field, it works this way too right? Even the set_value query

Reply:
Ah, thanks.